src/components/ContributorCard.tsx

```diff
@@ -34,12 +34,18 @@ export default function ContributorCard({ data }: { data: Card }) {
         className="rounded-full border"
       />
       <div className="mb-2 flex gap-12 text-2xl">
-        <Link href={data.html_url} target="_blank" className="duration-150 hover:text-[#411AFF]">
+        <Link href={data.html_url} target="_blank"                   
+                  aria-label="Visit us on GitHub"
+                  title="GitHub (External Link)"
+                  rel="noopener noreferrer" className="duration-150 hover:text-[#411AFF]">
           <FaGithub />
         </Link>
         <Link
           href={`https://github.com/WebXDAO/WebXDAO.github.io/commits?author=${data.login}`}
           target="_blank"
+          aria-label="Visit us on LaptopCode"
+          title="LaptopCode (External Link)"
```

Comment:
```suggestion
          title="Code Contributions (External Link)"
```

This redirects to the contributions in the repository, so `LaptopCode` is not appropriate